themes/socialbase/templates/node/links--node.html.twig

```diff
@@ -36,15 +36,19 @@
 
   {%- for key, item in links -%}
     {%- if item.link -%}
-    <div class="card__link">
-      {{ item.link }}
-    </div>
+      {% if style == 'sky' %}
+        <div class="card__link btn btn-default">
```

Comment:
We should make this part of the link attributes for both flavours. That will also fix a bug that is now made more prominent where the link click target is actually a lot smaller than the link/button's styling would suggest.